tools/deployment/make_linux_package.sh

```diff
@@ -51,7 +52,6 @@ DEBUG_PREFIX=$WORKING_DIR/debug
 
 function usage() {
   fatal "Usage: $0 -t deb|rpm -i REVISION -d DEPENDENCY_LIST
-
```

Comment:
I take a copy of the file from the project master in Sublime, edit, push up to my dev server, build test then commit. Why this process is causing lines to disappear is a mystery. Adding back in now.